src/azure-cli-core/azure/cli/core/style.py

```diff
@@ -26,6 +26,55 @@
 
 logger = get_logger(__name__)
 
+ERROR_INVALID_PARAMETER = 0x0057
+ENABLE_VIRTUAL_TERMINAL_PROCESSING = 0x0004
+
+
+# pylint: disable=import-error
+def set_conout_mode(new_mode, mask=0xffffffff):
```

Comment:
This is already in https://github.com/Azure/azure-cli/blob/a90d43973ba824773b1084061251856a212002bc/src/azure-cli/azure/cli/command_modules/resource/_win_vt.py#L54

I previously proposed to include it in core: https://github.com/Azure/azure-cli/pull/12942#discussion_r409968956